projects/step-frontend/src/lib/modules/plan-editor/directives/plan-editor-key-handler.directive.ts

```diff
@@ -136,7 +136,7 @@ export class PlanEditorKeyHandlerDirective {
     event: KeyboardEvent,
     isEmitByTreeOnly: boolean,
     keys: string | string[],
-    rights?: string | string[]
+    rights?: string | string[],
```

Comment:
nothing to do with dashboard

Reply:
@neogucky - I think prettier or something touched again random file. I never edited it. does it ever happened to you?